magnolia-voting-app/src/main/java/com/example/votingapp/utils/PollUtils.java

```diff
@@ -0,0 +1,42 @@
+package com.example.votingapp.utils;
+
+import java.util.*;
+
+import javax.jcr.*;
+
+import org.slf4j.*;
+
+import info.magnolia.jcr.*;
+
+public class PollUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PollUtils.class);
+
+    public static final String TO_DO = "To-do";
+    public static final String ON_GOING = "On-going";
+    public static final String DONE = "Done";
+
+    public static String pickStatus(Node source) throws RuntimeException {
```

Comment:
may be getPollStatus

Reply:
Originally, I named the method getPollStatus, but then I thought this maybe confuse with Java Bean naming convention. Besides, it is public static, we call it through PollUtils class so that I just named it pickStatus but it still make sense, PollUtils.pickStatus(...).
Or should I rename it pickPollStatus(...) ?